tools/src/vcf2genomicsdb_init.cc

```diff
@@ -715,10 +715,8 @@ std::set<std::string> process_samples(const std::string& sample_list, const std:
   std::vector<std::string> sample_files = TileDBUtils::get_files(samples_dir);
   for (auto sample_file: sample_files) {
     std::regex pattern("^.*(.vcf.gz$|.bcf.gz$)");
-    if (std::regex_search(sample_file, pattern)) {
-      if (sample_file.find("hdfs://") == std::string::npos) { 
-        sample_file = prefix+sample_file+suffix;
-      }
+    if (std::regex_search(sample_file, pattern)) { 
+      sample_file = prefix+sample_file+suffix;
```

Comment:
Can you do an `TileDBUtils::append_paths` here and remove the regex/prefix/suffix parts here?

Reply:
I can do that but isn't regex part is to determine the type of the file? Should we delete it?

Reply:
Does get_files() return the entire URL?

Reply:
No it doesn't but it returns path right? Doesn't path include file extension part?

Reply:
I am just thinking that `GenomicsDB` should not be the place to parse URIs, we should do it at the storage layer. Does that make sense?